bin/cheritest/purecap-mt/Makefile

```
@@ -0,0 +1,9 @@
+# $FreeBSD$
+
+.include <src.opts.mk>
+
+PROG=	cheritest-purecap-mt
+
+CHERI_THREAD_TESTS=	yes
+
+.include "../Makefile.purecap"
```

I think most places in the tree would use `${.CURDIR}`:
```suggestion
.include "${.CURDIR}/../Makefile.purecap"
```

---

In general, yes, but it seems this pattern is an exception (I did check before committing this):
```
jrtc4@zeno:~/cheri/cheribsd$ git grep '"\.\./Makefile' lib* *bin | wc -l
      85
jrtc4@zeno:~/cheri/cheribsd$ git grep '"\${\.CURDIR}/\.\./Makefile' lib* *bin | wc -l
      17
```

---

Huh, ok.